tests/feature/feature_console/deregister.pm

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2015-2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use registration;
+
+sub run() {
+    select_console 'root-console';
+
+    # check and list registered modules
+    my $out = script_output "SUSEConnect --status-text";
+    $out =~ /Registered/ or die "There's no module registered";
+
+    my $count = script_output("SUSEConnect --status-text | grep -c \"^\s*Registered\"", 100);
+    if ($count eq 0) {
+        die "There's no module registered";
+    }
+
+    # check if a certain module is registered, e.g. containers. OR: check all registered modules?
+    # FIXME my $all_reg_modules = script_output 'SUSEConnect --status-text | grep -B 2 '^\s*Registered' | grep '\/'';
+    my $module = "containers";
```

Comment:
just use one module. I suggest to crosscheck that the container module is part of "SCC_ADDONS", e.g.

```perl
die "pcm needs to be part of SCC_ADDONS for this test" unless check_var_array('SCC_ADDONS', $module);
```